com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zdo/ZdoCommandType.java

```diff
@@ -7,87 +7,7 @@
-import com.zsmartsystems.zigbee.zdo.command.CacheRequest;
-import com.zsmartsystems.zigbee.zdo.command.NetworkUpdateRequest;
-import com.zsmartsystems.zigbee.zdo.command.NetworkAddressResponse;
-import com.zsmartsystems.zigbee.zdo.command.IeeeAddressResponse;
-import com.zsmartsystems.zigbee.zdo.command.NodeDescriptorResponse;
-import com.zsmartsystems.zigbee.zdo.command.PowerDescriptorResponse;
-import com.zsmartsystems.zigbee.zdo.command.SimpleDescriptorResponse;
-import com.zsmartsystems.zigbee.zdo.command.ActiveEndpointsResponse;
-import com.zsmartsystems.zigbee.zdo.command.MatchDescriptorResponse;
-import com.zsmartsystems.zigbee.zdo.command.ComplexDescriptorResponse;
-import com.zsmartsystems.zigbee.zdo.command.UserDescriptorResponse;
-import com.zsmartsystems.zigbee.zdo.command.DiscoveryCacheResponse;
-import com.zsmartsystems.zigbee.zdo.command.UserDescriptorConf;
-import com.zsmartsystems.zigbee.zdo.command.DiscoveryStoreResponse;
-import com.zsmartsystems.zigbee.zdo.command.NodeDescriptorStoreResponse;
-import com.zsmartsystems.zigbee.zdo.command.PowerDescriptorStoreResponse;
-import com.zsmartsystems.zigbee.zdo.command.ActiveEndpointStoreResponse;
-import com.zsmartsystems.zigbee.zdo.command.SimpleDescriptorStoreResponse;
-import com.zsmartsystems.zigbee.zdo.command.RemoveNodeCache;
-import com.zsmartsystems.zigbee.zdo.command.FindNodeCacheResponse;
-import com.zsmartsystems.zigbee.zdo.command.ExtendedSimpleDescriptorResponse;
-import com.zsmartsystems.zigbee.zdo.command.ExtendedActiveEndpointResponse;
-import com.zsmartsystems.zigbee.zdo.command.EndDeviceBindResponse;
-import com.zsmartsystems.zigbee.zdo.command.BindResponse;
-import com.zsmartsystems.zigbee.zdo.command.UnbindResponse;
-import com.zsmartsystems.zigbee.zdo.command.BindRegisterResponse;
-import com.zsmartsystems.zigbee.zdo.command.ReplaceDeviceResponse;
-import com.zsmartsystems.zigbee.zdo.command.StoreBackupBindEntryResponse;
-import com.zsmartsystems.zigbee.zdo.command.RemoveBackupBindEntryResponse;
-import com.zsmartsystems.zigbee.zdo.command.BackupBindTableResponse;
-import com.zsmartsystems.zigbee.zdo.command.RecoverBindTableResponse;
-import com.zsmartsystems.zigbee.zdo.command.RecoverSourceBindResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementNetworkDiscoveryResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementLqiResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementRoutingResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementBindResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementLeaveResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementDirectJoinResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementPermitJoiningResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementCacheResponse;
-import com.zsmartsystems.zigbee.zdo.command.ManagementNetworkUpdateNotify;
+import com.zsmartsystems.zigbee.zdo.command.*;
```

Comment:
Is it possible to use the longer notation please - ie define each class. I thought that the code formatter should already do that and it avoids changing this code back and forward. As this is autocoded, it should have already been generated like this?

Reply:
Fixed